crates/agentgateway/src/http/localratelimit.rs

```diff
@@ -11,14 +11,23 @@ use crate::*;
 pub struct RateLimit {
 	ratelimit: Arc<ratelimit::Ratelimiter>,
```

Comment:
I think we can just do the normal `derive(Serialize)` and add `#[serde(skip_serializing)]`